src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -187,6 +188,12 @@ private void SetRemoteHostBitness()
                         WellKnownExperimentNames.RoslynOOP64bit);
                 }
 
+                if (!x64)
+                {
+                    // Use 64-bit OOP for systems with 32GiB+ memory
```

Comment:
Why 32?

Reply:
I've seen the 64-bit OOP use over 16GiB while working on Roslyn, but never more than 24.

Reply:
probably want to use experiment service to do this A/B testing?